web/server/vue-cli/src/views/ProductDetail.vue

```diff
@@ -1,8 +1,6 @@
 <template>
   <v-container id="product" fluid>
-    <keep-alive>
```

Comment:
If you use keep-alive, it will caches the components states. It should be handled properly, when a cached component is showed but something is changed in the URL. It was easier to remove it and initialize it properly when the component is mounted again.